ui/shared/components/panel/variants/Frequencies.jsx

```diff
@@ -182,72 +199,126 @@ const POPULATIONS = [
     queryParams: { [GENOME_VERSION_37]: 'dataset=gnomad_sv_r2_1' },
     helpMessage: GNOMAD_SV_CRITERIA_MESSAGE,
   },
+]
+
+const MITO_POPULATIONS = [
+  {
+    ...CALLSET_POP,
+    titleContainer: sectionTitle,
+    section: HOM_SECTION,
+  },
+  {
+    field: 'callset_heteroplasmy',
+    fieldTitle: 'This Callset',
+    acDisplay: 'AC',
+    titleContainer: sectionTitle,
+    section: HET_SECTION,
+  },
   {
     field: 'gnomad_mito',
     fieldTitle: 'gnomAD mito',
+    titleContainer: sectionTitle,
+    section: HOM_SECTION,
+    precision: 3,
+    ...GNOMAD_URL_INFO,
+  },
+  {
+    field: 'gnomad_mito_heteroplasmy',
+    fieldTitle: 'gnomAD mito',
+    titleContainer: sectionTitle,
+    section: HET_SECTION,
     precision: 3,
-    urls: { [GENOME_VERSION_38]: 'gnomad.broadinstitute.org' },
-    queryParams: { [GENOME_VERSION_38]: 'dataset=gnomad_r3' },
+    ...GNOMAD_URL_INFO,
   },
   {
     field: 'helix',
     fieldTitle: 'Helix mito',
+    titleContainer: sectionTitle,
+    section: HOM_SECTION,
+    precision: 3,
+  },
+  {
+    field: 'helix_heteroplasmy',
+    fieldTitle: 'Helix mito',
+    titleContainer: sectionTitle,
+    section: HET_SECTION,
     precision: 3,
   },
 ]
 
+const DETAIL_SECTIONS = [
+  {
+    name: 'Global AFs',
+    hasDetail: pop => pop && pop.filter_af && (pop.filter_af !== pop.af),
+    display: () => [{ subTitle: '', value: 'af' }],
```

Comment:
Its not a blocking change, but needing to set `subTitle: ''` everywhere adds unnecessary overhead for maintaining these and adding new sections. It would be better if the rendering could handle adding a default so we only need to define subtitle when its actually defined